tutordiscovery/plugin.py

```diff
@@ -32,6 +32,8 @@
         "OAUTH2_KEY_SSO_DEV": "discovery-sso-dev",
         "CACHE_REDIS_DB": "{{ OPENEDX_CACHE_REDIS_DB }}",
         "EXTRA_PIP_REQUIREMENTS": [],
+        "REPOSITORY": "https://github.com/openedx/course-discovery.git",
+        "REPOSITORY_VERSION": "{{ OPENEDX_COMMON_VERSION }}",
```

Comment:
As discussed, it would be great if we would document these settings in the README. Since the README has no such docs for now, you can choose to do that in a separate PR.